### ISSUE TITLE
Make breadcrumb a nav landmark

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#839]
Part of #820

- [x] Change
   `<div class="al-search-breadcrumb">` to 
   `<nav class="al-search-breadcrumb" role="navigation" aria-label="breadcrumb">`
